client/shipping-label/views/purchase/steps/packages/index.js

```diff
@@ -106,11 +140,10 @@ const PackagesStep = ( {
 	return (
 		<StepContainer
 			title={ __( 'Packages' ) }
-			isSuccess={ isValidWeight && isValidPackages }
-			isError={ ! isValidWeight || ! isValidPackages }
-			summary={ renderSummary() }
+			{ ...getContainerState() }
 			expanded={ expanded }
 			toggleStep={ () => labelActions.toggleStep( 'packages' ) } >
+			{ renderWarning() }
```

Comment:
When some piece of HTML is gonna render conditionally, I usually prefer to put that condition here, like this:
```{ ! hasAnyPackagesConfigured && renderWarning() }```
Or something similar with the ternary operator, instead of having to go into the function to check if it always render something.

Personal preference, feel free to ignore :)